.circleci/config.yml

```diff
@@ -0,0 +1,186 @@
+      - checkout
+      - run: |
+          mv client/app client/test /home/weave/
+          cd /home/weave; yarn run lint
+          cd /home/weave; yarn test
+
+  xplatform-build:
+    <<: *defaults
+    steps:
+      - checkout
+      - run: GOARCH=arm  make BUILD_IN_CONTAINER=false GO_BUILD_INSTALL_DEPS= prog/scope
+      - run: GOOS=darwin make BUILD_IN_CONTAINER=false GO_BUILD_INSTALL_DEPS= prog/scope
+
+  build:
+    <<: *defaults
+    steps:
+      - checkout
+      - setup_remote_docker
+      - attach_workspace:
+          at: .
+      - run: make BUILD_IN_CONTAINER=false SUDO= static all
+      - run: cd extras; make BUILD_IN_CONTAINER=false
+      - run: make -C tools/runner
+      - persist_to_workspace:
+          root: .
+          paths:
+          - scope.tar
+          - cloud-agent.tar
+          - tools/runner/runner
+          - prog/externalui/
+          - prog/staticui/
+          - report/report.codecgen.go
+          - render/detailed/detailed.codecgen.go
+
+  integration-tests:
+    machine:
+      image: circleci/classic:201709-01
+    working_directory: /home/circleci/src/github.com/weaveworks/scope
+    environment:
+      CIRCLE_ARTIFACTS: /tmp/artifacts
+      CLOUDSDK_CORE_DISABLE_PROMPTS: 1
+      GOPATH: /home/circleci/
+    parallelism: 2
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run: |
+          sudo apt-get update
+          sudo apt-get install python-pip jq pv
+          sudo pip install awscli
+      - run: mkdir $CIRCLE_ARTIFACTS
+      # kick off creation of test VMs
+      - run: test -z "$SECRET_PASSWORD" || bin/setup-circleci-secrets "$SECRET_PASSWORD"
+      - run: test -z "$SECRET_PASSWORD" || (cd integration; ./gce.sh make_template)
+      - run: test -z "$SECRET_PASSWORD" || (cd integration; ./gce.sh setup && eval $(./gce.sh hosts); ./setup.sh)
+      - run: make deps; touch tools/runner/runner
+      # Run all integration tests
+      - run:
+          command: test -z "$SECRET_PASSWORD" || (cd integration; eval $(./gce.sh hosts); ./run_all.sh)
+          no_output_timeout: 5m
+      # Destroy testing VMs:
+      - run:
+          command: test -z "$SECRET_PASSWORD" || (cd integration; ./gce.sh destroy)
+          background: true
+      # Code coverage
+      - run: ./tools/cover/gather_coverage.sh ./coverage
+      - run: goveralls -repotoken $COVERALLS_REPO_TOKEN -coverprofile=profile.cov -service=circleci
+      - run: cp coverage.* */*.codecgen.go $CIRCLE_ARTIFACTS
+      - store_artifacts:
+          path: /tmp/artifacts
+
+  deploy:
+    <<: *defaults
+    environment:
+      IMAGES: scope cloud-agent
+    steps:
+      - checkout
+      - setup_remote_docker
+      - attach_workspace:
+          at: .
+      # PATH must be set here as circle currently does not support string interpolation
+      # for environment
+      # https://discuss.circleci.com/t/bin-sh-mkdir-command-not-found/8710
+      - run: export PATH="$PATH:$HOME/.local/bin"
```

Comment:
I think this `export` might not work: https://circleci.com/docs/2.0/env-vars/#using-bash_env-to-set-environment-variables

Reply:
Removed the line and comment.